server/graphql/v2/input/ExpenseAttachedFileInput.ts

```diff
@@ -0,0 +1,18 @@
+import { GraphQLString, GraphQLInputObjectType, GraphQLNonNull } from 'graphql';
+
+/**
+ * To create or edit an optional expense file attachment
+ */
+export const ExpenseAttachedFileInput = new GraphQLInputObjectType({
+  name: 'ExpenseAttachedFileInput',
+  fields: {
+    id: {
+      type: GraphQLString,
+      description: 'ID of the file',
+    },
+    url: {
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
Maybe should we have considered a URL scalar for that.

Reply:
We can introduce that, I'll do it as a separate PR tho because `ExpenseItems` have an `url` field that needs to be updated too.